ruby-challenges.rb

```diff
@@ -10,7 +10,16 @@
 num3 = 221
 # Expected output: '221 is odd'
 
-
+def even_odd(num)
+    if num % 2 === 0
+    p 'even'       
```

Comment:
To get the expected output, please also consider the whole expected output. For instance, '221 is odd'